### DISCUSSION TITLE
Organiser's Guide

### DISCUSSION THREAD

**Discussion by gavincampbell** [dataplat/DataSaturdays#166] [Ideas]
I like this idea: https://devopsdays.org/organizing/, it sets out pretty clearly what the "rules" are for using their brand. Not advocating re-using their rules, just their transparency in communicating said rules.

**Comment by stephlocke** [comment]
You can also check out knowledgebase.satrdays.org

**Comment by way0utwest** [comment]
I do think we need some rules, but I would hope this contains more guidelines than requirements.

**Comment by gavincampbell** [comment]
Yes, I imagine most of the knowledgebase would be guidelines. The rules are more about what the criteria are to be an "official" DataSaturdays - or whatever name it is -  event. 

Sample rule from DevOpsDays:
_These are community events, so your event must have an open call for proposals and accept registrations from the general public. Internal devops events focused on a specific organization or curated events with all speakers privately selected are wonderful and we encourage them, but they won’t be listed as “devopsdays” events on this site._

There is also some "guidance", which includes things like "you probably shouldn't spend the sponsor money on organising committee meetings in Bora-Bora."

**Comment by rwlpf** [comment]
Hopefully, this is the best place for this question/comment.  As someone who tried and failed to get a SQL Saturday event organised.  I had a look at the organiser's guide and got a bit of the "bunny in the headlights" feeling.  They are great, and having one would be a fantastic resource.  What I thought would be good to see is mentors who are willing to work alongside those that might need some help. (yes Jen Stirrup was helping me, I did not see the wood for the trees).  Based on other events I have work with to organise there are a lot of tasks, the order and detail are not easy to see until you have done one.  An experienced eye can often see what needs to be done next, and what can be left till later.

**Comment by Avahey** [comment]
I think the guide to a successful event would be a good addition. It could be a living document that's modified by people who organize the events. Maybe hosted here on github so pr's can be reviewed by a group of people and then approved and published to the site? This is how we do our developer aspirations at my company.

**Comment by amtwo** [comment]
I think this would be a good candidate to host on the Wiki portion of this Repo. That would give the flexibility to have both a complete "First Timer's Guide to Organizing" that helps organize dependencies & decisions you'll need to make as an organizer, as well as cover some of the targeted FAQ & How To articles. 

I don't believe GitHub allows for accepting PRs directly into the Wiki, but I'm sure there's a workflow to manage that deployment.

[a comment by an automated account is not shown]

**Comment by suxstellino** [comment]
I think that every event has its own story. I've organised SQL Saturdays since 2014 and I didn't (sometimes couldn't) followed half of the guide. This is due for the majority of cases to cultural differences, regulations differences, and so on. I know that in Italy sometimes we are chaotic people 😂 but, to tell you the truth, the requirements are somehow different. 

What about approaching on a self made guide at least for each Country? Well, we can start from a, let's say, starting wiki from which we can inherit the "local" version.

[a comment by an automated account is not shown]

**Comment by ghotz** [comment]
I agree with @way0utwest and @suxstellino things should be like guidelines, not prescriptive as much as possible. Bureaucracy and rules were major problems with previous SQL Sats IMO. While there's no doubt about a common branding benefits,  organizing an event is not that hard these days and quite a few UGs already started doing them their own way.... and, please, keep it simple! :-)

[a comment by an automated account is not shown]
